broadcast-message.php

```diff
@@ -27,14 +27,19 @@
 
 require_once('common.php');
 
-authentication_require_role('global-admin');
+if ($CONF['sendmail_only_global_admin'] == 'YES' ) {
```

Comment:
so, if you're upgrading and haven't added sendmail_only_global_admin to your config then any admin can then use this? 

( i think you need to flip the logic and have a config lg for granting permission to normal admins, so by default it remains a global admin only feature)